plugins/source/aws/scripts/example_queries/main.go

```diff
@@ -123,7 +125,7 @@ type query struct {
 func (q query) Simplify() (string, error) {
 	qs := q.query
 	qs = regexp.MustCompile(`[ \t]+`).ReplaceAllString(qs, " ")
-	qs = regexp.MustCompile(`(?i)insert into aws_policy_results`).ReplaceAllString(qs, "")
+	qs = regexp.MustCompile(`(?i)insert into `+policyResultsTable).ReplaceAllString(qs, "")
```

Comment:
Just refactored this to use a constant so that it's not hidden somewhere deep in the file